### PR TITLE
fix: protect against NPE

### PR DESCRIPTION
**Pull request by sdelamo**
close: https://github.com/micronaut-projects/micronaut-aws/issues/1084

### CODE REVIEW

File: function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautAwsProxyRequest.java [hunk 1]
```diff
@@ -103,11 +104,31 @@
...
             setAttribute("micronaut.AUTHENTICATION", securityContext.getUserPrincipal());
         }
     }
 
+    /**
+     *
+     * @param securityContext Security Context
+     * @return returns false if the security context is not present, the associated event is null or the event's request context is null
+     */
+    public static boolean isSecurityContextPresent(@Nullable SecurityContext securityContext) {
```
Comment (ilopmar) [inline function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautAwsProxyRequest.java:117]: Do you need this method to be `public`?

Comment (sdelamo) [reply function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautAwsProxyRequest.java:117]: For the test mainly.

Comment (ilopmar) [reply function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautAwsProxyRequest.java:117]: Then make it package private

Comment (sdelamo) [reply function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautAwsProxyRequest.java]: done

**Review by ilopmar — APPROVED** [review]

**Pull request merged**